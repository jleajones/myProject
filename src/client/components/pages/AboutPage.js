import React from 'react';
import { Helmet } from 'react-helmet';

const AboutPage = () => {
  return (
    <div>
      <Helmet>
        <title>About Page</title>
        <meta name="description" content="This is the About..." />
      </Helmet>
      <div>this is the about page..</div>
    </div>
  );
};

export default AboutPage;
