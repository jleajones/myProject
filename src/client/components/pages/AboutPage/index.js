import React from 'react';
import { Helmet } from 'react-helmet';

import Page from '../../shared/Page'

const AboutPage = () => {
  return (
    <Page name='AboutPage'>
      <Helmet>
        <title>About Page</title>
        <meta name="description" content="This is the About..." />
      </Helmet>
      <div>this is the about page..</div>
    </Page>
  );
};

export default AboutPage;
