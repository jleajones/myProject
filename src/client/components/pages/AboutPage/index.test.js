import React from 'react';
import { shallow } from 'enzyme';
import AboutPage from '.';

describe('<AboutPage />', () => {
  test('Renders component without crashing', () => {
    const component = shallow(<AboutPage />);

    expect(component).toMatchSnapshot();
  });
});
