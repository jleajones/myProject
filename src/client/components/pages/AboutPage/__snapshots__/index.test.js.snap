// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage /> Renders component without crashing 1`] = `
<Connect(Component)
  pageName="AboutPage"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      About Page
    </title>
    <meta
      content="This is the About..."
      name="description"
    />
  </HelmetWrapper>
  <div>
    this is the about page..
  </div>
</Connect(Component)>
`;
